lib/OnyxUtils.ts

```diff
@@ -248,78 +252,54 @@ function getAllKeys() {
         return cache.getAllKeys();
     });
 
-    return cache.captureTask(taskName, promise);
+    return cache.captureTask(taskName, promise) as Promise<Set<OnyxKey>>;
```

Comment:
Why did we cast this to `Promise<Set<OnyxKey>>`, isn't the right type `Promise<OnyxValue<OnyxKey>>`?

Reply:
Ideally we should fix the types in my opinion, and if for some reason that is not possible, I think we should leave a comment on why it is ok to cast here